### DISCUSSION TITLE
Which version of Prometheus is compatible with Kiali 1.64.0?

### DISCUSSION THREAD

**Discussion by Yajnalatha** [kiali/kiali#5983] [Q&A]
Which version of Prometheus is compatible with Kiali 1.64.0?

**Comment by jmazzitelli** [comment]
That's a good question :) We typically test the majority of the time with the version that ships with Istio's Prometheus addon.

- https://istio.io/latest/docs/ops/integrations/prometheus/
- https://github.com/istio/istio/tree/master/manifests/addons

I don't think the Prom query API that Kiali uses changes much, so I would be surprised if there is a version Kiali doesn't support. Best bet is to try it and see. But I think the official answer would be "whatever version Istio supports, Kiali supports".
